src/fisher.jl

```diff
@@ -36,11 +36,37 @@ immutable FisherExactTest <: HypothesisTest
 
     # conditional maximum likehood estimate of odd ratio
     ω::Float64
+end
 
-    function FisherExactTest(a::Int, b::Int, c::Int, d::Int)
-        ω = cond_mle_odds_ratio(a, b, c, d)
-        new(a, b, c, d, ω)
-    end
+"""
+    FisherExactTest(a::Integer, b::Integer, c::Integer, d::Integer)
+
+Perform Fisher's exact test of the null hypothesis that the success probabilities ``a/c``
+and ``b/d`` are equal, that is the odds ratio ``(a/c) / (b/d)`` is one, against the
+alternative hypothesis that they are not equal.
+
+The contingency table is structured as:
+
+| -  | X1 | X2 |
+|:--:|:--:|:--:|
+|*Y1*| a  | b  |
+|*Y2*| c  | d  |
+
+!!! note
+    The print output contains the conditional maximum likelihood estimate of the odd-ratio
```

Comment:
"odds ratio" (also fix other occurrences below). Also, "print" would better be "show", with a link to the function.